source/Trashbot.js

```diff
@@ -18,14 +19,22 @@ export default class Trashbot extends Pixi.Sprite {
         if (this.position.x + this.width < 0) {
             this.position.x = Reference.GAME_WIDTH
         }
+        var killedBy
         game.children.forEach((toCompare) => {
```

Comment:
Changed. Went with "child"

Also got rid of those unnecessary comment slashes.